### ISSUE TITLE
External IP on gke

### ISSUE THREAD

**Issue by christopherL91** [istio/issues#21]
I'm trying to get the external (client ip) out from the istio proxy. I have modified the istio-auth.yaml file to include the following lines for the ingress: 
```
  annotations:
    service.beta.kubernetes.io/external-traffic: "OnlyLocal"
```
Digging deeper shows that envoy has a flag called ```use_remote_address``` that actually uses the x-forwarded-for header, the default value for this flag is false. Is there some way to change this?

**Comment by kyessenov** [comment]
We'll add it to ingress config template. Unfortunately the entire config is generated so custom templates are not easy to bring in.

**Comment by christopherL91** [comment]
Thank you! Will you be able to fix this in the upcoming release?

**Comment by kyessenov** [comment]
We'll have a major beta release in several months, but I'll link a PR when this gets fixed on master.

**Comment by kyessenov** [comment]
was merged as https://github.com/istio/pilot/pull/831, released in istio 0.1 alpha series.
